tests/filecheck/transforms/set-memory-space.mlir

```diff
@@ -0,0 +1,42 @@
+// RUN: ./compiler/snax-opt %s -p set-memory-space --allow-unregistered-dialect --print-op-generic | filecheck %s
```

Comment:
Why is --allow-unregistered-dialect necessary here, which operations are not registered??